datafusion/core/src/physical_optimizer/sort_enforcement.rs

```diff
@@ -2605,51 +2623,6 @@ mod tests {
         Ok(())
     }
 
-    #[tokio::test]
-    // With new change in SortEnforcement EnforceSorting->EnforceDistribution->EnforceSorting
```

Comment:
What is the reason to remove this test? This PR's description implies that `EnforceDistribution` is now idempotent so I would expect this test to continue to work as before 🤔

Reply:
It is leftover during development. I re-added this test. Thanks for pointing this out. I works as you expect.